client/src/RatingsReviews/Ratings/Bar.jsx

```diff
@@ -35,35 +36,25 @@ const InnerBar = styled.div`
   background-color: #1c1c1c;
 `;
 
-class Bar extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      resultsFilter: {}
-    }
-    this.setRatingFilter = this.setRatingFilter.bind(this);
-  }
-
-  setRatingFilter() {
-    this.props.setRatingFilter(this.props.star);
-  }
-
-  render() {
-    return (
-      <BarContainer>
-        <StarCount>{this.props.star}</StarCount>
-        <OuterBar onClick={this.setRatingFilter}><InnerBar width={this.props.percentage} /></OuterBar>
-      </BarContainer>
-    );
-  }
+function Bar(props) {
```

Comment:
I think it is a matter of preference but destructuring props may make code cleaner!

Reply:
totally. will get to that